apps/accounts/views.py

```diff
@@ -279,6 +283,26 @@ class Steps(Enum):
         Steps.PREVIEW.value: "provider_registration/preview.html",
     }
 
+    def log_creation(
```

Comment:
This allows us to log various events to the django log_entries table, from _outside_ the admin.

It's used outside approve() method for a provider request because access to the user in a given request is required.

If we created a "robot" user, we coud use this same approach for logging automated updates too, but that's outside the scope of this PR.